README.md

```diff
@@ -28,11 +28,11 @@ No. It should work with any other environment as long as:
 
 **[1]** We hope to make this configurable in the future.
 
-# How should I use it?
+## How should I use it?
 
-## Software requirements
+### Software requirements
 
-- Python3
+- Python 3
```

Comment:
Does it work with all possible Python 3 versions? Python 3.1 differs from Python 3.11, but both fulfill the requirements.

Reply:
I would assume all currently supported Python 3 versions (e.g. > 3.8).

Reply:
3.6 should work as well, as it's what we have in Leap 15.5, and in the infra we use to deploy, it's what SUSE Linux Enterprise 5.5 has as well

```
jenkins-worker:~ # python3 -V
Python 3.6.15

Reply:
Requirements are quite important, so it shall be better specified.